examples/networking/router_interface/base-router-network.tf

```diff
@@ -0,0 +1 @@
+../network/datasource/main.tf
```

Comment:
why do we need this?

Reply:
it was originally used by port. example

Reply:
base-router-network -> base-external-network